localstack/services/awslambda/invocation/lambda_models.py

```diff
@@ -44,8 +44,9 @@
     "python3.8": "python:3.8@sha256:b3402a5f5e9535ba4787a1fd6b0ee39738dee18bdff861a0589571ba74122d35",
     "python3.9": "python:3.9@sha256:5b3585b121e6fb9707abb52c1f99cbab51939fee0769752ab6c641f20f479cf6",
     "nodejs12.x": "nodejs:12@sha256:16431b8d5eb26e80b37a80297ad67f70478c045339395bbd32f45091275ebb50",
```

Comment:
Any reason for not updating nodejs12.x? (newest sha would be `1389b8cc6bbd321f22b0218233da11bd49382bbee7689dba41c7377c12df65a5`)

Reply:
The snapshots did not change, but we can update it as well!